src/backend/distributed/commands/multi_copy.c

```diff
@@ -2139,7 +2139,7 @@ ShardIntervalListHasLocalPlacements(List *shardIntervalList)
 	ShardInterval *shardInterval = NULL;
 	foreach_ptr(shardInterval, shardIntervalList)
 	{
-		if (FindShardPlacementOnGroup(localGroupId, shardInterval->shardId) != NULL)
+		if (ActiveShardPlacementOnGroup(localGroupId, shardInterval->shardId) != NULL)
```

Comment:
I haven't added a test for this one. It seemed hard to do. @onderkalaci maybe you have an idea.

Reply:
I think that is not possible with the regular steps because local placements for MX cannot have replication > 1 and local placements for single node cannot be marked as inactive (I think?)

A synthetic way of doing it:
```SQL
select master_add_node('localhost', 5432, groupid:=0);
select master_add_node('localhost', 9700);

SET citus.shard_replication_factor TO 2;
update pg_dist_node set shouldhaveshards = true;
CREATE TABLE t1 (A int);
SELECT create_distributed_table('t1','a');

-- simulate a synthetic failure on the coordinator 
 update pg_dist_shard_placement set shardstate = 3 where nodeport = 5432;

-- now local placements on the coordinator has invalid placements so the function should return false
```